packages/babel-core/src/transformation/file/index.js

```diff
@@ -568,9 +582,19 @@ export default class File extends Store {
     let result: BabelFileResult = { ast };
     if (!opts.code) return this.makeResult(result);
 
+    let gen = generate;
+    if (opts.generatorOpts.recast) {
+      try {
+        let recast = require("recast");
+        gen = recast.generator || recast.print;
+      } catch (e) {
+        throw new Error(`Couldn't find generator ${JSON.stringify(generatorPath)}`);
```

Comment:
Oh I removed that since I just hardcoded recast - so this try/catch should be removed